app/views/report/explorer.html.haml

```diff
@@ -12,7 +13,9 @@
         = javascript_hide("menu_div1")
         = javascript_show("menu_div3")
   - else
+    setTimeout(function() {
```

Comment:
how about adding defer ot the javascript script instead? (see https://www.w3schools.com/tags/att_script_defer.asp)

Reply:
@ohadlevy that doesn't help, still failing as it requires the angular component to be ready. I have a proper fix in my head but that's for a longer run...